README.md

```diff
@@ -145,8 +145,8 @@ update existing keys in the database.
 Each "op" above corresponds to a write of a single key/value pair.
 I.e., a random write benchmark goes at approximately 400,000 writes per second.
 
-Each "fillsync" operation costs much less (0.3 millisecond)
-than a disk seek (typically 10 milliseconds).  We suspect that this is
+Each "file sync" operation costs much less (0.3 milliseconds)
+then a disk seeks (typically 10 milliseconds).  We suspect that this is
```

Comment:
... fillsync operation costs much less ... **than** a disk **seek** is the correct grammar as we are comparing two fillsync to disk seek times.